include/can/core/ids.hpp

```diff
@@ -27,7 +27,9 @@ enum class NodeId : uint8_t {
     pipette = 0x20,
     gantry_x = 0x30,
     gantry_y = 0x40,
-    head = 0x50,
+    head_left = 0x50,
```

Comment:
also a nitpick but if we swap these we can keep it (somewhat) backwards compatible. And I think it might be better to use the low nibble here - we're reserving all this space in the node id, might as well use it (I think we are, anyway - is it valid to have that low nibble be anything other than 0?)
```suggestion
    head = 0x50,
    head_right = 0x51,
    head_left = 0x52
```